runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorResponse.java

```diff
@@ -0,0 +1,24 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import io.netty.buffer.ByteBuf;
+import org.corfudb.protocols.wireprotocol.ICorfuPayload;
+
+/**
+ * Created by Maithem on 10/25/17.
+ */
+
+public class OrchestratorResponse implements ICorfuPayload<OrchestratorResponse> {
+
+    public OrchestratorResponse() {
+
+    }
+
+    public OrchestratorResponse(ByteBuf buf) {
+
+    }
+
+    @Override
+    public void doSerialize(ByteBuf buf) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)